var/spack/repos/builtin/packages/lbann/package.py

```diff
@@ -33,31 +33,38 @@ class Lbann(CMakePackage, CudaPackage):
     version('0.92', sha256='9187c5bcbc562c2828fe619d53884ab80afb1bcd627a817edb935b80affe7b84')
     version('0.91', sha256='b69f470829f434f266119a33695592f74802cff4b76b37022db00ab32de322f5')
 
-    variant('opencv', default=True, description='Builds with support for image processing routines with OpenCV')
-    variant('seq_init', default=False, description='Force serial initialization of weight matrices.')
-    variant('dtype', default='float',
-            description='Type for floating point representation of weights',
-            values=('float', 'double'))
+    variant('al', default=True, description='Builds with support for Aluminum Library')
     variant('build_type', default='Release',
             description='The build type to build',
             values=('Debug', 'Release'))
-    variant('al', default=True, description='Builds with support for Aluminum Library')
     variant('conduit', default=True,
             description='Builds with support for Conduit Library '
             '(note that for v0.99 conduit is required)')
-    variant('half', default=False,
-            description='Builds with support for FP16 precision data types')
+    variant('deterministic', default=False,
+            description='Builds with support for deterministic execution')
     variant('dihydrogen', default=False,
             description='Builds with support for DiHydrogen Tensor Library')
     variant('distconv', default=False,
             description='Builds with support for spatial, filter, or channel '
             'distributed convolutions')
-
-    variant('vtune', default=False, description='Builds with support for Intel VTune')
     variant('docs', default=False, description='Builds with support for building documentation')
+    variant('dtype', default='float',
+            description='Type for floating point representation of weights',
+            values=('float', 'double'))
     variant('extras', default=False, description='Add python modules for LBANN related tools')
+    variant('fft', default=False, description='Support for FFT operations')
+    variant('half', default=False,
+            description='Builds with support for FP16 precision data types')
+    variant('nvprof', default=False, description='Build with region annotations for NVPROF')
+    variant('opencv', default=True,
+            description='Builds with support for image processing with OpenCV')
+    variant('vtune', default=False, description='Builds with support for Intel VTune')
+    variant('unit_testing', default=False, description='Builds with support for unit testing')
```

Comment:
Just to make sure, this variant changes what gets installed, right? It doesn't just build unit tests that can be used at build-time?

Reply:
Still looking for an answer to this.

Reply:
It turns on a dependence for catch and builds the catch test, so I have removed the variant and tied into the `--test` flag.